CHANGES.md

```diff
@@ -1,5 +1,122 @@
 # Changes
 
+## stable-2.10.0
+
+This release introduces Linkerd extensions. The default control plane no longer
+includes Prometheus, Grafana, the dashboard, or several other components that
+previously shipped by default.  This results in a much smaller and simpler set
+of core functionality.  Visibility and metrics functionality is now available
```

Comment:
Want to use plural here since it's a set of functionalities.

```suggestion
of core functionalities.  Visibility and metrics functionality is now available
```

I think it should also be "Visibility and metrics functionalities are now..", but that stands out less to me.